web/modules/custom/joinup_notification/src/EventSubscriber/CommunityContentSubscriber.php

```diff
@@ -249,32 +257,55 @@ protected function generateArguments(EntityInterface $entity) {
 
     $arguments['@actor:full_name'] = $actor_first_name . ' ' . $actor_last_name;
     $arguments['@transition:motivation'] = $motivation;
+    $arguments['@entity:hasPublished:status'] = $this->hasPublished ? 'an update of the' : 'a new';
```

Comment:
This is OK since Joinup will never be translated to a different language than English, but in a standard multilingual site this wouldn't be good.